applications/nrf5340_audio/src/audio/audio_system.h

```diff
@@ -12,51 +12,51 @@
 #include <stdint.h>
 
 /**
- * @brief Toggle a test tone on and off
+ * @brief Toggle a test tone on and off.
  *
- * @param[in] freq Desired frequency of tone, 0 = off
+ * @param[in] freq Desired frequency of tone. Off if set to 0.
  *
- * @note A stream must already be running to use this feature
+ * @note A stream must already be running to use this feature.
  *
- * @return 0 on success, and -ENOMEM if the frequency is too low (buffer overflow)
+ * @return 0 on success; -ENOMEM if the frequency is too low (buffer overflow).
```

Comment:
I guess this could use retval:
```suggestion
 * @retval   -ENOMEM  The frequency is too low (buffer overflow).
 * @retval   0        Success 
```